packages/zosfiles/src/methods/create/Create.ts

```diff
@@ -134,15 +134,22 @@ export class Create {
         }
     }
 
-    public static async dataSetLike(session: AbstractSession, dataSetName: string, likeDataSetName: string): Promise<IZosFilesResponse> {
+    public static async dataSetLike(session: AbstractSession,
+                                    dataSetName: string,
+                                    likeDataSetName: string,
+                                    options?: Partial<ICreateDataSetOptions>): Promise<IZosFilesResponse> {
         // Required
         ImperativeExpect.toNotBeNullOrUndefined(dataSetName, ZosFilesMessages.missingDatasetName.message);
         ImperativeExpect.toNotBeNullOrUndefined(likeDataSetName, ZosFilesMessages.missingDatasetLikeName.message);
 
+        // Removes undefined properties
+        const tempOptions = !isNullOrUndefined(options) ? JSON.parse(JSON.stringify({ like: likeDataSetName, ...(options || {}) })) : {};
+        Create.dataSetValidateOptions(tempOptions);
```

Comment:
The `Create.dataSetValidateOptions` API doesn't recognize "like" as a valid option. Please add a case for "like" (e.g., `case "like":`) to the `switch` statement below:
https://github.com/zowe/zowe-cli/blob/7c16e6587628ce86f26ecc60bd64356df4ac4a02/packages/zosfiles/src/methods/create/Create.ts#L296-L301